lib/mayaHydra/flowViewportAPIExamples/footPrintNode/mhFootPrintNode.cpp

```diff
@@ -567,6 +567,20 @@ MBoundingBox MhFootPrint::boundingBox() const
 
 void* MhFootPrint::creator()
 {
+    int	isMayaHydraLoaded = false;
+    // Validate that the gpuCache plugin is loaded.
+    MGlobal::executeCommand( "pluginInfo -query -loaded mayaHydra", isMayaHydraLoaded );
+	if( ! isMayaHydraLoaded){
+        MStatus status;
+	    MString errorString = MStringResource::getString(rMayaHydraNotLoadedStringError, status);	     
+        if (! status){
+            status.perror("Cannot retrieve the rMayaHydraNotLoadedStringError string, but you need to load mayaHydra before creating this node");
+        }else{
+	        MGlobal::displayError(errorString);	    
```

Comment:
Strange that we'd have 2 ways to report errors.

Reply:
They are 2 different errors, one because we didn't find the localized string, the other because maya hydra is not loaded.

Reply:
Right, but why do we use MGlobal::displayError() in one case, and MStatus::perror() in the other?

Reply:
I removed one of them.